### ISSUE TITLE
Vi is not added to the list of editors

### ISSUE THREAD

**Issue by karansingh1559** [coala/coala#5239]
Coala does not recognise vi as an editor, although it does recognise vim and nvim.
Vi needs to be added to the list of editors in [OpenEditorAction.py](https://github.com/coala/coala/blob/master/coalib/results/result_actions/OpenEditorAction.py)
This is what happens when you try to open a file with vi:
 
Please enter a value for the parameter 'editor' (The editor to open the file with.): vi

> choose vi editor

The editor "vi" is unknown to coala. Files won't be opened at the correct positions and other quirks might occur. Consider opening an issue at https://github.com/coala/coala/issues so we can add support for this editor. Supported editors are: atom, emacs, emacsclient, geany, gedit, gvim, kate, nano, nvim, subl, vim, xed

**Comment by karansingh1559** [comment]
@Makman2 If this is a valid issue, please assign this to me :D

**Comment by RaiVaibhav** [comment]
Add Atom also in `gui`

**Comment by karansingh1559** [comment]
@RaiVaibhav atom is already there :)

**Comment by aramadsanar** [comment]
Hi @karansingh1559, can you assign this to me? I would like to work on this issue :)

Thanks

**Comment by jayvdb** [comment]
why cant you use `vim` ?

`vi` is a different editor.

**Comment by Naveenaidu** [comment]
@aramadsanar Only maintainers can assign issues. Please ask @Makman2 to assign the issue to you. :')

**Comment by sangamcse** [comment]
@jayvdb, I feel that `vi` should also be an option.
> `vi` is a different editor.

**Comment by Makman2** [comment]
I feel that too :+1:

**Comment by sangamcse** [comment]
```
vi --version
VIM - Vi IMproved 8.0 (2016 Sep 12, compiled Nov 29 2017 18:37:46)
Included patches: 1-503, 505-680, 682-1283
Compiled by root@apple.com
Normal version without GUI.
```
In my system, it is a package which was updated recently with MacOS 10.13.2 Developer Beta update.

**Comment by jayvdb** [comment]
right.  That is vim, not vi.  You also have a executable called `vim` I assume.  Most systems now alias vi to vim, or have a `vi` which is a slightly cut down version of `vim`.  Someone using vim should specify they are using vim as their editor.

If we want to help people who are using vim masquerading as vi, and dont know what to call it, we could have a feature which informs them to use `vim`.  That is different from creating support for a `vi` editor which is not comforming to the POSIX spec for `vi`
